apps/ledger-live-mobile/src/logic/firmwareUpdate.ts

```diff
@@ -4,8 +4,9 @@ import { satisfies as versionSatisfies } from "semver";
 
 const deviceVersionRangesForUpdate: { [key in DeviceModelId]?: string } = {
   nanoS: ">=1.6.1",
-  nanoX: ">=1.2.4-6",
+  nanoX: ">=1.2.4-6 || =2.1.0-lo2 || =2.1.0-lo4",
   nanoSP: ">=1.0.0-0",
+  nanoFTS: "=2.0.2-il0 || =2.0.2-il1 || =2.0.2-il2 || =2.0.2-il3",
```

Comment:
Why can't we rely on the flags from device info? [Looking at the specs from the confluence page](https://ledgerhq.atlassian.net/wiki/spaces/TA/pages/edit-v2/3560375295) it would seem like regardless of the model, regardless of the version, since we have the device info we can check for the language flag 🤔 

Instead of this ugliness

Reply:
This one specifically is for FirmwareUpdate, not for localization. We don't have flags for firmware update.
Concerning localization we still need to know the firmware version in order to decide to extract or not the subsequent language id, it's not as the other flags, it's an actual data that comes in a specific order.

That being said, I only use the version check in order to extract or not the language id. In the live layer itself I never check the version but rather the existence of a language id indeed.

Reply:
Gotcha